nautobot/utilities/tests/test_filters.py

```diff
@@ -1061,28 +1135,29 @@ def get_filterset_count(self, params, klass=None):
     def test_default_icontains(self):
         """Test a default search for an "icontains" value."""
         params = {"q": "Test Site"}
-        self.assertEqual(self.get_filterset_count(params), 4)
+        self.assertEqual(self.get_filterset_count(params), self.queryset.filter(name__icontains="Test Site").count())
```

Comment:
not blocking but would be good to change all of these to `assertQuerysetEqualAndNotEmpty`